openreview/stages/venue_stages.py

```diff
@@ -293,6 +295,20 @@ def get_content(self, api_version='1', conference=None, venue_id=None):
                     }
                 }
 
+            if self.commitments_venue:
+                content['paper_link'] = {
+                    'value': {
+                        'param': {
+                            'type': 'string',
+                            'regex': '(http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+)',
+                            'mismatchError': 'must be a valid link to an OpenrReview submission: https://openreview.net/forum?id=...'
+                        }
+                    },
+                    'description': 'Please provide the link to your ARR submission.',
```

Comment:
Can "The link should have the following format: https://openreview.net/forum?id=<PAPER_ID>" be added to the description?

Reply:
Does <PAPER_ID> make sense?